Indicators/ForceIndex.cs

```diff
@@ -0,0 +1,100 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    ///The Force Index is calculated by comparing the current market price with the previous market price 
+    ///and multiplying its difference with the traded volume during a specific time period.
+    ///
+    /// </summary>
+    public class ForceIndex : TradeBarIndicator, IIndicatorWarmUpPeriodProvider
+    {
+        private TradeBar _previousInput;
+
+        /// <summary>This indicator is used to smooth the ForceIndex computation</summary>
```

Comment:
Summary tags are usually in their own lines, see other xml docs

Reply:
Fixed by https://github.com/QuantConnect/Lean/pull/8155/commits/1ecec6636fb83897701ec0c55a938f6591f077a9